packages/realm/RealmJS.podspec

```diff
@@ -43,18 +43,12 @@ Pod::Spec.new do |s|
 
   s.source_files           = 'react-native/ios/RealmReact/*.mm'
   s.public_header_files    = 'react-native/ios/RealmReact/*.h'
-
+  s.resource_bundles       = { 'RealmJS' => ['PrivacyInfo.xcprivacy'] }
+  
   s.frameworks             = uses_frameworks ? ['React'] : []
-
   s.library                = 'c++', 'z', 'compression'
 
   s.pod_target_xcconfig    = {
-                                # Setting up clang
-                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
-                                'CLANG_CXX_LIBRARY' => 'libc++',
-                                # Setting the current project version and versioning system to get a symbol for analytics
-                                'CURRENT_PROJECT_VERSION' => s.version,
-                                'VERSIONING_SYSTEM' => 'apple-generic',
```

Comment:
What's the reason for this being removed?

Reply:
Lines 56 and 57 are problematic as they generates an additional file which makes the bundle with an incorrect structure (a file outside the bundle). The new privacy policies don't allow it. The additional files were used by the old analytics submission code, and as we don't need them, I find it simpler to remove the generation than figuring out how to make them fulfil the privacy policies.

The two compiler flags (lines 53 and 54) are not really needed anymore.

Reply:
Thanks for the explanation!